web/src/pages/people/index.tsx

```diff
@@ -72,16 +106,40 @@ export default function PeoplePage() {
         }
       />
 
-      {!isTreOpsStaff && (
-        <div className={styles["button-container"]}>
-          {isAdmin && <ApprovedResearcherImport />}
-          {(isAdmin || isIAO) && <ExternalInvite />}
-        </div>
-      )}
+      <div className={styles["button-container"]}>
+        {(isAdmin || isTreOpsStaff) && (
+          <form className={styles["search-container"]}>
+            <Input
+              placeholder="search users..."
+              id={styles.search}
+              name="search"
+              onChange={handleInputChange}
+              value={searchTerm}
+              aria-label="search users of the portal"
+            ></Input>
+            <Button
+              variant="tertiary"
+              icon={<SearchIcon />}
+              onClick={handleUserSearchSubmit}
+              type="submit"
+              aria-label="submit user search query"
+            ></Button>
+            {searchErrorMessage && (
+              <Alert type="error">
+                <AlertMessage>{errorMessage}</AlertMessage>
+              </Alert>
+            )}
```

Comment:
When I typed in something that didn't work (e.g. 'jh') I got a red cross on the search bar, but there was no accompanying error message to tell me what I did wrong. Also, if I then subsequently made a new search for something that did work (e.g. 'portal') the red cross never goes away.

Reply:
https://github.com/ucl-arc-tre/portal/pull/251/commits/045cc2464802aba82c28db257ec209f4a65cd326

shouldn't get the error now, only came up if the term was shorter than 3 characters, so probably an entra limitation